packages/core/src/components/EmptyState/MissingAnnotationEmptyState.tsx

```diff
@@ -37,7 +38,16 @@ type Props = {
   annotation: string;
 };
 
+const useStyles = makeStyles<BackstageTheme>(theme => ({
+  code: {
```

Comment:
How come you had to do this custom styling here? I would have hoped that the CodeSnippet were fine for use directly; is there something we can do to it instead so that it fits in better here as well as in other places?

Reply:
I wanted to give it a white background but the default is some shade of very light green.

Reply:
@freben fine with this?